### PR TITLE
Adjust ServoMortor sample

### PR DESCRIPTION
**Pull request by Ellerbach**
## Description

Adjust ServoMortor sample

## Motivation and Context

- Adjust ServoMortor sample

## Types of changes

- [x] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (work on Unit Tests, has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/main/CONTRIBUTING.md) document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: devices/ServoMotor/samples/Program.cs [hunk 1]
```diff
@@ -4,15 +4,16 @@
...
 using System.Diagnostics;
+using System.Threading;
 using Iot.Device.ServoMotor;
 using nanoFramework.Hardware.Esp32;
 
 Debug.WriteLine("Hello Servo Motor!");
 
 // When using an ESP32, you have to setup the pin function then create the PWM channel
-Configuration.SetPinFunction(21, DeviceFunction.PWM1);
+Configuration.SetPinFunction(5, DeviceFunction.PWM1);
```
Comment (josesimoes) [inline devices/ServoMotor/samples/Program.cs:14]: Maybe add to the readme the same block that I added yesterday in BMXX80 about running on a ESP32...

Comment (Ellerbach) [reply devices/ServoMotor/samples/Program.cs:14]: It's in the readme already!

Comment (Ellerbach) [reply devices/ServoMotor/samples/Program.cs:14]: My plan is to review all samples and everything for a consistent copy/paste of comment in a kind of batch. Also I need to adjust all the README. Need a long evening for that.

Comment (josesimoes) [reply devices/ServoMotor/samples/Program.cs:14]: My bad! Missed that! 😊

**Pull request merged**